frontends/web/src/routes/settings/manage-accounts.module.css

```diff
@@ -21,20 +38,38 @@
 .acccountLink {
     align-items: center;
     display: flex;
-    flex-basis: calc(100% - 180px);
-    flex-grow: 0;
+    flex-basis: calc(100% - 100px);
```

Comment:
should give more space for long account names on small screen, I'll add to the commit message.